content/assignments/minishell/src/utils.h

```diff
@@ -0,0 +1,31 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#ifndef _UTILS_H
+#define _UTILS_H
+
+#include "../util/parser/parser.h"
+
+
+/* Useful macro for handling error codes. */
+#define DIE(assertion, call_description)	\
+	do {																		\
+		if (assertion) {											\
+			fprintf(stderr, "(%s, %s, %d): ",		\
+				__FILE__, __func__, __LINE__);		\
+			perror(call_description);						\
+			exit(EXIT_FAILURE);									\
+		}																			\
```

Comment:
```suggestion
#define DIE(assertion, call_description)			\
	do							\
		if (assertion) 					\
			fprintf(stderr, "(%s, %s, %d): ",	\
				__FILE__, __func__, __LINE__);	\
			perror(call_description);		\
			exit(EXIT_FAILURE);			\
		}						\
```
Change your tab width to 8 before adding tabs. Or just commit my suggestion and squash it locally.